docs/src/pages/reference/useQuery.md

```diff
@@ -43,6 +43,7 @@ const {
   refetchOnReconnect,
   refetchOnWindowFocus,
   retry,
+  retryOnMount
```

Comment:
Can you add a comma here?

Reply:
Ah good catch. Yup. Revision coming out soon.